pom.xml

```diff
@@ -20,6 +20,8 @@
         <jenkins.baseline>2.504</jenkins.baseline>
         <jenkins.version>${jenkins.baseline}.1</jenkins.version>
         <hpi.compatibleSinceVersion>6.8.1</hpi.compatibleSinceVersion>
+        <hpi.strictBundledArtifacts>true</hpi.strictBundledArtifacts>
+        <hpi.bundledArtifacts>kubernetes-client,kubernetes-client-api,kubernetes-httpclient-okhttp,kubernetes-model-admissionregistration,kubernetes-model-apiextensions,kubernetes-model-apps,kubernetes-model-autoscaling,kubernetes-model-batch,kubernetes-model-certificates,kubernetes-model-common,kubernetes-model-coordination,kubernetes-model-core,kubernetes-model-discovery,kubernetes-model-events,kubernetes-model-extensions,kubernetes-model-flowcontrol,kubernetes-model-gatewayapi,kubernetes-model-metrics,kubernetes-model-networking,kubernetes-model-node,kubernetes-model-policy,kubernetes-model-rbac,kubernetes-model-resource,kubernetes-model-scheduling,kubernetes-model-storageclass,openshift-client,openshift-client-api,openshift-model,openshift-model-autoscaling,openshift-model-config,openshift-model-console,openshift-model-hive,openshift-model-installer,openshift-model-machine,openshift-model-machineconfiguration,openshift-model-miscellaneous,openshift-model-monitoring,openshift-model-operator,openshift-model-operatorhub,openshift-model-storageversionmigrator,openshift-model-tuned,openshift-model-whereabouts,snakeyaml-engine,volumesnapshot-client,volumesnapshot-model,zjsonpatch</hpi.bundledArtifacts>
```

Comment:
Most of this looks ok, but `snakeyaml-engine` seems surprising given https://plugins.jenkins.io/snakeyaml-api/, but maybe it's not part of SnakeYaml core. I'll review this list.

Reply:
Looking at https://github.com/snakeyaml/snakeyaml?tab=readme-ov-file#overview, it seems like `snakeyaml` is for YAML 1.1, but `snakeyaml-engine` is for YAML 1.2. As seen in https://github.com/jenkinsci/snakeyaml-api-plugin/pull/137, currently the API plugin only bundles `snakeyaml`, which I guess makes sense because these artifact appear to be totally independent. I guess if it ever becomes a problem, a new `snakeyaml-engine` API plugin could be created, but at least for now there don't appear to be any explicit dependencies on that artifact from other plugins: https://github.com/search?type=code&q=+owner%3Ajenkinsci+snakeyaml-engine.

`volumesnapshot-client,volumesnapshot-model,zjsonpatch` are all fabric8 libraries.